Sources/NIOSSL/SSLCallbacks.swift

```diff
@@ -130,6 +130,22 @@ extension KeyLogCallbackManager {
     }
 }
 
+/// PSK Identity Callback Manager
+public typealias NIOPSKClientIdentityCallback = (NIOSSLSecureBytes?) -> (key: NIOSSLSecureBytes, identity: NIOSSLSecureBytes)
+public typealias NIOPSKServerIdentityCallback = (NIOSSLSecureBytes?, NIOSSLSecureBytes?) -> (key: NIOSSLSecureBytes, identity: NIOSSLSecureBytes)
```

Comment:
Relatedly, only the keys need to be in `NIOSSLSecureBytes`: the rest can be regular `String` objects.

Reply:
Resolved in 409e1c1